libbeat/publisher/broker/membroker/broker.go

```diff
@@ -219,9 +219,9 @@ func (b *Broker) eventLoop() {
 			events = b.events
 		}
 
-		b.logger.Debug("active events: ", activeEvents)
+		// b.logger.Debug("active events: ", activeEvents)
```

Comment:
yeah, loads of debugs commented out in the fast path. I did still keep them, so I can re-enabled them If I really need them.